test/test_nn.py

```diff
@@ -12366,6 +12366,14 @@ def test_embedding_padding_idx(self, device, dtype):
         self.assertEqual(output[0][2].sum(), 0)
         self.assertEqual(output[1][1].sum(), 0)
 
+        # change padding vector
```

Comment:
In addition to this test, I think it'd be good to add to the `test_embedding_functional` test that compares functional and module forms.

In particular, the test should assert that, given the same `padding_idx` and `weight`, the module and functional outputs should *differ* when the embedding vector at `padding_idx` for the module is modified from its all-zero initialization.

The functional form should return zeros for `padding_idx` and the module form should return the modified embedding value.